eos-tech-support/eos-enable-coredumps

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
```

Comment:
I’d ideally like to see this written in Python (because yeuch, bash), but this is sufficiently short. (Don’t rewrite it in Python unless you’re a fanatic.)

It’s missing a copyright header.

Reply:
I'm not a fanatic and I also like python, but I think this still qualifies as "simple enough for bash" :)

Will add the header